tests/testthat/test-renderer1-facet-lines.R

```diff
@@ -46,8 +46,12 @@ wb.facets <-
        first=list(year=1975, country=c("United States", "Vietnam")),
        selector.types=list(country="multiple"),
        title="World Bank data (multiple selection, facets)")
+custom_delay <- function(milliseconds) {
+  Sys.sleep(milliseconds / 1000)
+}
```

Comment:
please undo addition of custom_delay function and instead just use Sys.sleep (no benefit for helper function if it is only used once)

Reply:
done